app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -39,7 +39,7 @@
     <string name="network_error">网络错误</string>
     <plurals name="videos">
         <item quantity="one">视频</item>
-        <item quantity="other"></item>
+        <item quantity="other"/>
```

Comment:
Why are you deleting and not translating ?

Reply:
I don't know either, the file was automatically written by Weblate.